lib/exceptions/fail_build_exception.dart

```diff
@@ -0,0 +1,33 @@
+/// Exception thrown when Ozzie fails a set of integration tests
+class FailBuildException implements Exception {
+  String message;
+
+  FailBuildException(this.message);
+
+  /// Helper to build an exception based on warnings in the report
+  factory FailBuildException.onWarning(String featureName) {
+    return FailBuildException("""
+¯\_(ツ)_/¯  Warnign detected
+
+Ozzie has measured warnings on the $featureName feature, and your configuration
```

Comment:
nit but might want to reword to `Ozzie has **detected** warnings...`
Also typo on the next like `warnigns`

Reply:
🤦‍♂